pip/pep425tags.py

```diff
@@ -127,6 +127,54 @@ def get_platform():
     return distutils.util.get_platform().replace('.', '_').replace('-', '_')
 
 
+def is_manylinux1_compatible():
+    # Only Linux, and only x86-64 / i686
+    if get_platform() not in ("linux_x86_64", "linux_i686"):
+        return False
+
+    # "wide" Unicode mode is mandatory (always true on CPython 3.3+)
+    if sys.maxunicode <= 0xFFFF:
+        return False
+
+    # Check for presence of _manylinux module
+    try:
+        import _manylinux
+        return bool(_manylinux.manylinux1_compatible)
```

Comment:
Yes, `_manylinux` is basically a config file that may be added to python by a distributor, as a way of providing information about a particular python install to pip. It's not an actual module with, like, code in it. So vendoring it would defeat the purpose :-). See PEP 513 for details.